src/check.h.in

```diff
@@ -170,6 +170,41 @@ CK_DLL_EXP void CK_EXPORT suite_add_tcase(Suite * s, TCase * tc);
 CK_DLL_EXP TCase *CK_EXPORT tcase_create(const char *name);
 
 /**
+ * Associate a test case with certain tags.
+ * Replaces any existing tags with the new set.
+ *
+ * @param tc the test case
+ *
+ * @param tags string containing arbitrary tags separated by spaces
+ *        This will be copied. Passing NULL clears all tags. 
+ *
+ * @since 0.11.0
+ * */
+CK_DLL_EXP void CK_EXPORT tcase_set_tags(TCase * tc,
+					 const char *tags);
+/**
+ * Get tags currently associated with a test case.
+ *
+ * @param tc the test case
+ *
+ * @param tags pointer to array in which to return space separated tag
+ * string.
+ *
+ * @param max_sz maximum size of tags array including null terminator.
+ * If max_sz is passed as 0 then nothing will be  written to tags and it
+ * may not be terminated.
+ *
+ * @return length it would like to have written excluding the
+ * terminating null. NOTE: this may be equal or longer to max_sz, in
+ * which case you know that the returned string truncated. (See
+ * snprintf).
+ *
+ * @since 0.11.0
+ * */
+CK_DLL_EXP int CK_EXPORT tcase_get_tags(TCase * tc,
```

Comment:
Its certainly not absolutely necessary.

It could be useful if anybody has a use case for inserting extra tags (or removing existing tags) against tests that had already been tagged.  Its probably not that likely but there's also a slight benefit when it comes to the unit tests as I can isolate testing of set/get from the test runner functionality.

If you would prefer it removed I'm fine with that but let me know.